digitalpublicgoods/kiva-protocol.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "Kiva Protocol",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://github.com/kiva/aries-guardianship-agency/blob/master/LICENSE"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://protocol-docs.web.app/"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "No"
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "National Civil Registration Authority Act"
```

Comment:
Should specify which country, as I suspect it is "National" in a given country. Sierra Leone, I presume?

Reply:
Resolved in commit cf9fde6